mountpoint-s3-crt-sys/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+* Update to latest CRT dependencies
+* Default exponential backoff value retries is now 500ms rather than 25ms ([awslabs/aws-c-io#612](https://github.com/awslabs/aws-c-io/pull/612))
```

Comment:
We should probably avoid trying to maintain our own changelog for the entire CRT. I think we should only use this crate's changelog for changes in how we _package_ the CRT, things like changing the build process or the configuration.

Reply:
Ah, makes sense! I see all changelog entries for that crate are of that nature.